### ISSUE TITLE
The date and time window is too large

### ISSUE THREAD

**Issue by psknick** [Stellarium/stellarium#2722]
Hello!
In version 1.22.3, the size of the Date and Time window turned out to be disproportionately large. This size of the window makes it difficult to work with the image of the starry sky. If possible, please add the ability to customize the size and font size of this window. Also, if possible, add the function of disabling the Julian days tab.
Thank you!

**Comment by alex-w** [comment]
> Also, if possible, add the function of disabling the Julian days tab.

Why you want it?

**Comment by psknick** [comment]
For me, the Julian day tab is not relevant and it takes up space on the screen.

Now I have returned version 0.22.2 and in version 1.22.3 all windows have become larger and this is very unusual.

Maybe I think so, but in windows version 1.22.3 there are problems with fonts - they seem not sharp. In version 0.22.2 there are no problems with sharpness.

**Comment by alex-w** [comment]
Hmm... Please try 0.22.3 also.

**Comment by psknick** [comment]
I tried ... Very strange! Such differences... The first photo is 1.22.3. Second photo 0.22.3.
I have Windows 11 full license with all updates. Monitor 24 inches.

Very unusual look Qt6...

If possible, add at least the ability to customize the size and font size of the Date and Time window. This window constantly sticks out on the screen and the new version takes up a lot of space on the screen.
Thank you!

![2](https://user-images.githubusercontent.com/44753018/193915882-17b5526a-e055-4d58-9166-a78fc2a5f520.png)
![1](https://user-images.githubusercontent.com/44753018/193915891-22dffa97-1959-43c0-82be-fd0949283357.png)

**Comment by 10110111** [comment]
Looks like high-DPI scaling is on. The difference between Qt5 and Qt6 is that the latter supports fractional scaling factors, while the former rounds them by default.

Let me guess: you have 125% scaling set in Windows display settings, right?

**Comment by psknick** [comment]
Yes, indeed 121%. Otherwise, everything is too small on the monitor. And what should I do?

**Comment by 10110111** [comment]
Somehow you need to pass `QT_SCREEN_SCALE_FACTORS=1` environment variable to Stellarium. I need some time to learn how to do it in Windows.

**Comment by 10110111** [comment]
OK, that variable doesn't appear to work for some reason. Anyway, here's what you can do:

Open _Properties_ of the shortcut you use to start Stellarium. Change the path field from e.g. 
```
"C:\Program Files\Stellarium\Stellarium.exe" -platform windows:altgr
```
to the following:
```
c:\windows\system32\cmd.exe /c "set QT_ENABLE_HIGHDPI_SCALING=0 && start ^"c:\Program Files\Stellarium^" stellarium.exe  -platform windows:altgr"
```
The point is to start Stellarium with `QT_ENABLE_HIGHDPI_SCALING=0` environment variable set.

**Comment by psknick** [comment]
Thank you very much! In version 1.22.3, everything started to look just fine!

**Comment by alex-w** [comment]
> The point is to start Stellarium with `QT_ENABLE_HIGHDPI_SCALING=0` environment variable set.

@gzotti @10110111 Maybe adding another startup shortcut can help here?

**Comment by 10110111** [comment]
IMHO there are already too many of them. It would be better to have a GUI already to choose from the kludges.

**Comment by gzotti** [comment]
I have an optional (usually hidden) extension `[gui]/flag_font_selection` to select the main font. We could extend this to add a submenu dialog for "Font, screen scaling, (other tweaks?)". Settings should be stored immediately. The environment variable will likely need a program restart, but Ruslan probably found where to set them early enough in the start sequence.

**Comment by psknick** [comment]
"[gui]/flag_font_selection"

I have customized the fonts. As a result, the size of the Date and Time window has decreased. Everything turned out very well. Thanks!
![2022-10-22_14-50](https://user-images.githubusercontent.com/44753018/197337517-d09af795-fe39-4ab6-8886-e6fda099a7f9.png)

**Comment by psknick** [comment]
"[gui]/flag_font_selection"
This setting is very, very useful - please enable it by default in the next releases. Thanks!

**Comment by 10110111** [comment]
Fixed by #3157.
